Change the introductory text to only talk about the designated IP. (#53)
ietf-wg-add/draft-ietf-add-ddr: draft-ietf-add-ddr.md

Resolver is designated by the originally provisioned resolver. "Designated" in
this context means that the resolvers are operated by the same entity or
cooperating entities; for example, the resolvers are accessible on the same

> IP address, or there is a certificate that claims ownership over the designated
> resolver IP address (regardless of what IP the designated resolver is on).

Reviewer: I think the cert covers the original "designating" IP address, not the "designated" IP address.
Since one method uses the name, I think this should either be:

```suggestion
IP address, or there is a certificate that claims ownership over the original
resolver IP address or hostname.
```
or:
```suggestion
IP address, or there is a certificate that claims ownership over the original
designating resolver.
```

Resolver is designated by the originally provisioned resolver. "Designated" in
this context means that the resolvers are operated by the same entity or
cooperating entities; for example, the resolvers are accessible on the same

> IP address, or there is a certificate that claims ownership over the original

Reviewer 2: Good catch Tommy.

We've lost the distinction of the ownership being over the IP address (as opposed to any other property of the designating resolver, such as a name). I suggest inserting "IP address of the" before "original"
